percy/sitemap.js

```diff
@@ -0,0 +1,37 @@
+const Axios = require("axios")
+const PercyScript = require("@percy/script");
+const parseUrl = require("url-parse")
+const xml = require("xml2json")
+
+async function loadSitemap(url) {
+  const response = await Axios.get(url, {
```

Comment:
Can you check if the sitemap is a local file or a remote URL? So we can pass a local file to limit the number of urls to parse.

Reply:
Yup, now loads from local file or URL.